Dockerfile

```diff
@@ -22,14 +23,21 @@ COPY . /doccano
 WORKDIR /doccano
 RUN tools/ci.sh
 
+ARG HADOLINT_VERSION=v1.17.1
+RUN curl -fsSL "https://github.com/hadolint/hadolint/releases/download/${HADOLINT_VERSION}/hadolint-Linux-$(uname -m)" -o /usr/local/bin/hadolint  \
```

Comment:
Currently the build fails because of `/bin/bash: curl: command not found`. If you move this snippet above `install-mssql.sh` everything should work.